docs/topics/assembly_building-and-deploying-the-booster-to-openshiftonline.adoc

```diff
@@ -10,12 +10,11 @@
 [#building-and-deploying-the-{parameter-mission}-booster-to-openshiftonline]
 = Building and Deploying the {parameter-mission-name} Booster to {OpenShiftOnline}
 
-You have two options for executing the {parameter-mission-name} booster on {OpenShiftOnline}:
+You have two options for executing the {parameter-mission-name} booster on {OpenShiftOnline}. Although both methods use the same `oc` commands to deploy your application, using {launcher-oso} provides an automated booster deployment workflow that executes the `oc` commands for you.
```

Comment:
Same as above, sentence between colon ":" and a bullet list.

Reply:
Implemented your suggestions. Moved the sentence after the 2 bullets.